aizynthtrain/modelling/expansion_policy/featurize.py

```diff
@@ -6,19 +6,21 @@
 from scipy import sparse
 from rdkit import Chem, DataStructs
 from rdkit.Chem import AllChem
+from rxnutils.data.batch_utils import read_csv_batch
```

Comment:
We should update rxnutils as well with batch_utils

Reply:
It was included in the latest release. (You were not included on that PR)